app/assets/v2/js/pages/fulfill_bounty.js

```diff
@@ -24,6 +24,13 @@ window.onload = function() {
 
     $('#submitBounty').validate({
       submitHandler: function(form) {
+        try{
+          bounty_address();
+        }catch(exception){
+          _alert('You are on an unsupported network.  Please change your network to a supported network.');
```

Comment:
Can we wrap these in `gettext()` ?

Reply:
255adaf2b8f0692fb02acc6311ed67fcfb1730f4